LEAF_Nexus/admin/templates/menu.tpl

```diff
@@ -1,4 +1,34 @@
     <a href="../" class="buttonNorm"><img src="../../libs/dynicons/?img=go-home.svg&amp;w=16" alt="Main Page" title="Main Page" />Main Page</a>
+    <div id="headerMenuHelp_container" style="display: inline-block">
+        <a id="button_showHelp" tabindex="0" class="buttonNorm" alt="Help Popup" title="Help" aria-haspopup="true" aria-expanded="false" role="button"><img style="vertical-align: sub;" src="../../libs/dynicons/?img=help-browser.svg&amp;w=16">&nbsp;Help</a>
+        <div id="headerMenu_help" tabindex="0">
+            For Help contact your primary admin:
+            <div id="help-primary-admin" style="font-weight:bold;">
+            <script type="text/javascript">
+                $.ajax({
+                    type: 'GET',
+                    url: "../api/system/primaryadmin",
+                    success: function(res) {
+                        let emailString = res['email'] != '' ? " - " + res['email'] : '';
+                        if(res["firstName"] !== undefined)
+                        {
+                            $('#help-primary-admin').html(res['firstName'] + " " + res['lastName'] + emailString);
+                        }
+                        else if(res["userName"] !== undefined)
+                        {
+                            $('#help-primary-admin').html(res['userName']);
+                        }
+                        else
+                        {
+                            $('#help-primary-admin').html('Primary Admin has not been set.');
+                        }
```

Comment:
Need fail catch.  Might even need it to be an "error" response to have them reload the page.

Reply:
Added fail catch, this would annoy users if they were prompted for that on a thing most users dont even hover over. Since its just a menu hover for help.